eng/common/scripts/Invoke-GitHubAPI.ps1

```diff
@@ -90,13 +90,29 @@ function Invoke-GitHubAPIGet {
   return $resp
 }
 
-function SplitMembers ($membersString)
+function Split-GitHubAPIParameterMembers ($membersString)
 {
   if (!$membersString) { return $null }
```

Comment:
Should we consider only returning $null if memberstring is null? and then return an array with one empty string if memberstring is empty? I think that might handle most of your special cases below.